kevm-pyk/src/kevm_pyk/kproj/evm-semantics/state-utils.md

```diff
@@ -187,270 +89,6 @@ The `"network"` key allows setting the fee schedule inside the test.
     rule #asScheduleString("ShanghaiToCancunAtTime15k") => CANCUN
 ```
 
-The `"rlp"` key loads the block information.
```

Comment:
Will we have a K structure like this, or will we trust the op-get to do this load of block info?